en/docs/apis/advanced-topics/connect-to-an-external-identity-provider.md

```diff
@@ -0,0 +1,146 @@
+# Connect to an External Identity Provider
+
+Choreo uses an in-built Identity Provider (IdP) by default to manage OAuth clients and generate tokens required to authenticate Choreo APIs. Choreo also grants organization administrator privileges to users to configure a required authorization server as an identity provider(IdP) via the Choreo Console. This allows you to [add one or more external identity providers](#add-an-external-identity-provider) to your Choreo organization and [obtain access tokens from the identity providers](#obtain-an-access-token-from-an-external-identity-provider) to exchange and get Choreo access tokens to invoke APIs. This capability lets you expose your APIs to users who are external to Choreo.
```

Comment:
@wasuradananjith, we are good to change "OAuth clients" to "OAuth 2.0 clients" right?
Also can you please have a look at the second and third comment here and confirm whether. we can go ahead with the suggestions?

Reply:
+1 to change "OAuth clients" to "OAuth 2.0 clients"

Reply:
> required -> external? or third-party? shall we give an example?

@Nirothipan any idea on this?

Reply:
> STS

Where is this STS word? Is it from the diagram?

Reply:
required authorization server as an identity provider(IdP) via the Choreo Console. > external authorization server as an identity provider(IdP) via the Choreo Console ( Eg:- OKTA)

Reply:
> > STS
> 
> Where is this STS word? Is it from the diagram?

No, it's not in the diagram. But what I meant is, should we refer to it as the STS, rather than only the Authorization server as well?

Reply:
STS is mote technical, so needed I geuss.

Reply:
Applied appropriate suggestions based on the comments.